FWCore/Concurrency/interface/WaitingTaskWithArenaHolder.h

```diff
@@ -24,9 +24,9 @@
 
 #include "tbb/task_arena.h"
 
-namespace edm {
+#include "FWCore/Concurrency/interface/WaitingTask.h"
```

Comment:
What required the include over the forward declaration?

Reply:
Likely something during the development, going back to forward declaration seems to compile fine now. Will go back.